homeassistant/components/fritz/strings.json

```diff
@@ -52,5 +52,35 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "external_ip": { "name": "External IP" },
+      "external_ipv6": { "name": "External IPv6" },
+      "device_uptime": { "name": "Device Uptime" },
+      "connection_uptime": { "name": "Connection Uptime" },
```

Comment:
I don't think this follows the entity naming convention. Only the first word should be capitalized.
`Connection Uptime` -> `Connection uptime`